android-components/components/support/ktx/src/main/res/values/strings.xml

```diff
@@ -5,13 +5,13 @@
   ~  file, You can obtain one at http://mozilla.org/MPL/2.0/.
   -->
 
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools" xmlns:moz="http://schemas.android.com/apk/res-auto">
     <!-- Text displayed when choosing which app to call with after selecting a phone number-->
     <string name="mozac_support_ktx_menu_call_with">Call with…</string>
     <!-- Text displayed when choosing which app to email with after selecting an email address-->
     <string name="mozac_support_ktx_menu_email_with">Email with…</string>
     <string name="mozac_support_ktx_menu_share_with">Share with…</string>
     <string name="mozac_support_ktx_share_dialog_title">Share via</string>
     <!-- Text for confirmation "snackbar" shown after copying an image to the clipboard. -->
-    <string name="mozac_support_ktx_snackbar_delegate_image_copied">Image copied to clipboard</string>
+    <string name="mozac_support_ktx_snackbar_delegate_image_copied" moz:removedIn="114" tools:ignore="UnusedResources">Image copied to clipboard</string>
```

Comment:
Why did we need to add this?

Reply:
To clarify a bit better, why not move this string to the ui-widget component as well? That way "Image copied to clipboard" can be used shared with both apps without it needing to be re-translated for each.

```xml
    <string name="mozac_ui_widget_image_copied">Image copied to clipboard</string>
```

Reply:
Agreed, moving it to ui-widget component makes sense 👍